src/tracks/gosling-track/gosling-track.ts

```diff
@@ -694,6 +713,37 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
                 }
             }
         }
+        /**
+         * Copied from HorizontalTiled1DPixiTrack
+         */
+        calculateZoomLevel() {
```

Comment:
Here is the original 
https://github.com/higlass/higlass/blob/335f4eea6445d07baa5eda2e66077dca461b7777/app/scripts/HorizontalTiled1DPixiTrack.js#L61